config/env/development.js

```diff
@@ -20,15 +20,5 @@ module.exports = {
   // models: {
   //   connection: 'someMongodbServer'
   // }
-  host: '127.0.0.1',
-  port: 1337,
-  google: {
```

Comment:
If we do this change development might stop. @ogzhnozlci @berkantesk  change your config/local.js according to this document.